src/NServiceBus.Core/Licensing/LicenseExpiredForm.cs

```diff
@@ -80,19 +84,62 @@ void browseButton_Click(object sender, EventArgs e)
 
         void PurchaseButton_Click(object sender, EventArgs e)
         {
-            Process.Start("http://particular.net/licensing");
+            Process.Start("https://particular.net/licensing");
         }
 
         void getTrialLicenseButton_Click(object sender, EventArgs e)
         {
-            if (CurrentLicense != null && CurrentLicense.IsTrialLicense)
+            var parameterCollection = new NameValueCollection
```

Comment:
Why use a `NameValueCollection`? Below then you have to do fancy things like:

`string.Join("&", parameterCollection.AllKeys.Select(key => $"{HttpUtility.UrlEncode(key)}={HttpUtility.UrlEncode(parameterCollection[key])}"))`

which is harder to read than

```
var parameters = new Dictionary<string, string> { ... };
string.Join("&", parameters.Select(kvp => $"{HttpUtility.UrlEncode(kvp.Key)}={HttpUtility.UrlEncode(kvp.Value)}"))
```